test/py/libvfio_user.py

```diff
@@ -124,6 +124,16 @@
 VFIO_DEVICE_STATE_V1_RESUMING = (1 << 2)
 VFIO_DEVICE_STATE_MASK = ((1 << 3) - 1)
 
+VFIO_DEVICE_FEATURE_MIGRATION = (1)
```

Comment:
I didn't check - does it make sense to add tests here too, or too early/tricky?

Reply:
nit: no need for parenthesis